### PR TITLE
Feat cml tmux

### PR DESCRIPTION
**Pull request by sgherdao**
Since we are it, time to stop procrastinating and submit that PR for tmux support :)

```
This commit introduce a new command 'cml tmux'
It allows users to connect to multiple node consoles using 'tmux'. 
It creates a new 'tmux' session with the lab title and the first 4 characters of the lab ID.

The `--group` option offers two grouping choices:
- 'panes': Group all connections within a single 'tmux' window. (default)
- 'windows': Represent each connection as an individual 'tmux' window.

Version 2 only.
```

### CODE REVIEW

File: virl/about.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.4.2"
+__version__ = "1.4.3"
```
Comment (xorrkaz) [inline virl/about.py:1]: Please do not bump version yourself.

File: requirements.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@ gunicorn
 idna
 itsdangerous
 Jinja2
+libtmux
```
Comment (xorrkaz) [inline requirements.txt:10]: Is this also required for testing?

**Review by xorrkaz — CHANGES_REQUESTED** [review]

File: virl/about.py (hunk 1 above)
Comment (sgherdao) [reply virl/about.py:1]: Sorry, I based my branch out of `master`, do you want me to change it to 1.4.2?

File: requirements.txt (hunk 2 above)
Comment (sgherdao) [reply requirements.txt:10]: right :)!

Comment (xorrkaz) [reply requirements.txt:10]: Then it needs to be in test-requirements.txt.

File: virl/about.py (hunk 1 above)
Comment (xorrkaz) [reply virl/about.py:1]: joe-dev should have 1.4.3.  I see it in my pull.

Comment (sgherdao) [reply virl/about.py:1]: So we good?

File: requirements.txt (hunk 2 above)
Comment (sgherdao) [reply requirements.txt:10]: Committed already!
https://github.com/CiscoDevNet/virlutils/pull/143/commits/2b9338537f368dffbaa9688d41ccfe985cb09e3c

**Review by xorrkaz — APPROVED** [review]

**Pull request merged**